service/src/test/resources/dynamicConfig.yaml

```diff
@@ -33,4 +33,11 @@
      celeborn.test.tenant.timeoutMs.only: 100s
      celeborn.test.tenant.enabled.only: false
      celeborn.test.tenant.int.only: 10
+     celeborn.client.push.queue.capacity: 1024
+   users:
+     - name: Jerry
```

Comment:
We could introduce `TENANT_USER` enum in `ConfigLevel`. This config level is implicitly expressed via `users` field in `FsConfigServiceImpl` and explicitly expressed via `level` column in `DbConfigServiceImpl`.